### ISSUE TITLE
Non-hyperspace capable ships making and waiting for jump...

### ISSUE THREAD

**Issue by PhoenixRiver** [endless-sky/endless-sky#496]
TLDR; Non-hyperspace capable ships shouldn't turn and wait to jump (when "j" is held) and should instead keep fighting previous targets/ships attacking my friendlies/ships attacking them...

So I was escorting a freighter and I had taken out some of the pirates attacking it but we _needed_ to go. My deployed fighters (I had no other ships in my fleet) promptly stopped attacking the pirates, turned and lined up ready to go with us while we were still under attack... we jumped without them of course, because they had no hyperspace drives (acceptable loss - but averted as I was able to go back for them) but we made it with only a hair's breadth, had they kept fighting I'd have been rather happier since they couldn't jump with us anyway.
This mechanic would allow the tactical leaving of a fighting force in a system as cover for a jump away or a return jump - handy in some systems and makes more common sense (why would fighter pilots perform jump manouvers when they can't! :)

**Comment by omagasohe** [comment]
That or they should break for the nearest planet. That would make even more sense. Seriously, I just told them to hole up and I'll be back, They should run like hell =P

**Comment by bobafetthotmail** [comment]
non-hyperspace capable ships with crew should attempt to dock while you are lining up for the jump, and remain behind only if there was not enough time to do so (you can give them more time by keeping j pressed).
Asking them to remain and fight by default isn't very humane.

Drones can stay and keep attacking unless recalled, that's fine.

**Comment by endless-sky** [comment]
This should be addressed in commit 263ec57: fighters that have no jump drive installed will keep fighting like a normal escort while you prepare to jump, then congregate at a planet they can land / refuel / repair on once you leave.

**Comment by PhoenixRiver** [comment]
> This should be addressed in commit 263ec57: fighters that have no jump drive
> installed will keep fighting like a normal escort while you prepare to jump,
> then congregate at a planet they can land / refuel / repair on once you
> leave.

Nice compromise :)

**Comment by endless-sky** [comment]
Fixed and released in 0.8.10.
